src/__tests__/role-helpers.js

```diff
@@ -78,13 +74,15 @@ function setup() {
     unnamedSection: getByTestId('a-section'),
     namedSection: getByTestId('named-section'),
     anchor: getByTestId('a-link'),
+    invalidAnchor: getByTestId('invalid-link'),
     h1: getByTestId('a-h1'),
     h2: getByTestId('a-h2'),
     h3: getByTestId('a-h3'),
+    unnamedImg: getByTestId('a-img-1'),
+    presentationImg: getByTestId('a-img-2'),
+    namedImg: getByTestId('a-img-3'),
     nav: getByTestId('a-nav'),
     article: getByTestId('a-article'),
-    menuItem: getByTestId('a-menuitem-1'),
-    menuItem2: getByTestId('a-menuitem-2'),
```

Comment:
Can we add new stuff at the end here? Makes the snapshot changes easier to review.